src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -1,175 +1,189 @@
+Rate limiting allows you to [limit the amount of events Sentry accepts](/product/accounts/quotas/#what-counts-toward-my-quota-table-view) per project for a defined time span. While this is quite useful for managing your monthly event quota, keep in mind that once a defined threshold is crossed, **subsequent events will be dropped**. Therefore, you shouldn't constantly be hitting your rate limit; rather, it should act as a ceiling intended to protect you from unexpected spikes.
 
-Under `[Project Settings] » Client Keys » Configure`, you can create multiple DSN keys per-project and assign different (or no) limits to each key. This will allow you to dynamically allocate keys (with varying thresholds) depending on Release, Environment, and so forth.
+In **[Project] > Settings > Client Keys (DSN)**, click "Configure", and you can create multiple DSN keys per project and assign different (or no) rate limits to each key. This will allow you to dynamically allocate keys (with varying thresholds) depending on release, environment, and so on.
 
 ![Per DSN Key rate limits](manage-event-stream-11.png)
 
-For more information see [What Counts Toward My Quota](/product/accounts/quotas/#what-counts-toward-my-quota-table-view)
+### Setting Useful Rate Limits {#-how-to-set-proper-rate-limits}
 
-## 7. Spike Protection
+A good way to set a project rate limit is by figuring out the expected event volume based on your average traffic. Let's look at an example:
 
-Sentry also applies a dynamic rate limit to your account designed to protect you from short-term spikes. However, we would recommend applying all of the previously mentioned methods. For more information see:
+![Calculating rate limits](manage-event-stream-14.png)
 
-- [Spike Protection](/product/accounts/quotas/#spike-protection)
-- [Protect Yourself Against Spikes in Events with Spike Protection](https://blog.sentry.io/2018/05/08/event-spike-protection#what-counts-as-a-spike)
+- Open the project DSN key configuration under **[Project] > Settings > Client Keys (DSN)** by clicking "Configure".
+- In the `KEY USAGE IN THE LAST 30 DAYS` graph and look for the highest point, or the maximum daily rate. In this example, the maximum daily rate in the last month is less than 326K.
+- Based on that, we can define a ceiling **daily** maximum value of approximantely 330K, which is around 13,750 events an **hour**, or  about 230 events a **minute**.
+- You can set a daily, hourly, or minute-based rate limit. We'd recommend using a minute-based rate to avoid situations where a random event spike might exhaust your daily or hourly set quota and leave you blind for a long while.
```

Comment:
```suggestion
- You can set a daily, hourly, or minute-based rate limit. We'd recommend using a minute-based rate to avoid situations where a random event spike might exhaust your daily or hourly quota and leave you blind for a long period.
```

🤷